src/js/themes/hpe.js

```diff
@@ -2441,7 +2441,7 @@ const buildTheme = (tokens, flags) => {
       content: {
         background: 'background-floating',
         border: { color: 'border-weak' },
-        margin: 'xxsmall',
+        margin: '3xsmall',
```

Comment:
@taysea updated this from 'xsmall' to '3xsmall'

Reply:
Looks good! -- can you adjust the changeset in this PR as well?

Reply:
Oops, yes I will update that too